packages/workbox-background-sync/src/Queue.ts

```diff
@@ -86,15 +107,15 @@ class Queue {
    *     for you (defaulting to `Date.now()`), but you can update it if you
    *     don't want particular requests to expire.
    */
-  async pushRequest(entry) {
+  async pushRequest(entry: QueueEntry) {
     if (process.env.NODE_ENV !== 'production') {
-      assert.isType(entry, 'object', {
+      assert!.isType(entry, 'object', {
```

Comment:
Just checking—this is an alternative to `assert && assert.is...()` that you've used in previous PRs?

Reply:
Hmmmm, I don't remember doing that, but looks like I did.

Yeah, the `!` operator is just a way of telling Typescript that you know a value in some context is definitely not `null` or `undefined`. In this case it's because we're in the `process.env.NODE_ENV !== 'production'` block.

I'll change those previous cases to the more idiomatic form in the next PR.